build/grunt.js

```diff
@@ -1,86 +1,26 @@
-import {gruntCustomizer, gruntOptionsMaker} from './options-customizer.js';
 import isDocsOnly from './docs-only.js';
 
 module.exports = function(grunt) {
   require('time-grunt')(grunt);
 
-  let _ = require('lodash');
```

Comment:
Other than here and in options-customizer, lodash isn't used anywhere else, I don't think.